src/components/FeatureViews/Search/SearchResults.js

```diff
@@ -140,163 +158,163 @@ const SearchResults = (props) => {
+                                                className="searchChip"
+                                                onClick={displayAll(SEARCH_ACCOUNTS)}
+                                            />
                                         </Grid>
-                                    ))
-                                    :
-                                    accounts.length > 0 && searchState.primaryFilter !== SEARCH_COURSES?
-                                        accounts.slice(start["account"], end["account"]).map((account) => (
+                                    }
+                                </Grid>
+                                <Grid container spacing={16} direction={"row"}>
+                                    {searchState.primaryFilter === SEARCH_ACCOUNTS ?
+                                        accounts.map((account) => (
                                             <Grid item
-                                                  key={account.user_id}
+                                                key={account.user_id}
                                                 sm={3}>
                                                 <AccountsCards user={account} key={account.user_id} />
                                             </Grid>
                                         ))
                                         :
-                                        ""}
+                                        accounts.length > 0 && searchState.primaryFilter !== SEARCH_COURSES ?
+                                            accounts.slice(start["account"], end["account"]).map((account) => (
+                                                <Grid item
+                                                    key={account.user_id}
+                                                    sm={3}>
+                                                    <AccountsCards user={account} key={account.user_id} />
+                                                </Grid>
+                                            ))
+                                            :
+                                            ""}
+                                </Grid>
+                                {accounts.length > 4 ?
+                                    <div className={"results-nav"}>
+                                        {
+                                            <IconButton disabled={lessResults !== 1}
+                                                className={"less"}
+                                                onClick={displayResults("less", SEARCH_ACCOUNTS)}>
+                                                <LessResultsIcon />
+                                            </IconButton>
+                                        }
+                                        {searchState.params.account.accountPage}
+                                        {
+                                            <IconButton disabled={moreResults}
```

Comment:
instead of using a state variable for this, and you setting the state variable in the beginning, just call the comparison that you used above, but condense it. For example`<IconButton disabled={ maxPage > currentPage } ... />` and `<IconButton disabled={maxPage >= currentPage && currentPage !== 1}.../>` 

You can do this by setting only 1 state variable - `maxPage` - and deleting the other two you had - `moreResults` and `lessResults`. 

Doing this has 2 benefits:
1. Reduces the amount of code it takes to achieve this
2. Less code = more maintainable code